README.md

```diff
@@ -1,93 +1,51 @@
-<h1 align="center">
-    <img alt="Gnonce" title="Gnonce" src="https://github.com/Gnonce/gatsby-starter-gnonce/blob/master/src/images/icon.png" width="200"> </br>
-    Gatsby Starter Gnonce
-</h1>
+# Safe Landing Page
 
-<h4 align="center">
-  Gatsby + Typescript + Styled Components = 💖
-</h4>
+![license](https://img.shields.io/github/license/gnosis/safe-landing-page)
+![release](https://img.shields.io/github/v/release/gnosis/safe-landing-page)
 
-<p align="center"><img src="	https://img.shields.io/maintenance/yes/2019.svg"> <a target="_blank" href="https://gatsby-starter-gnonce.netlify.com/"><img src="https://img.shields.io/website-up-down-green-red/https/gatsby-starter-gnonce.netlify.com.svg"></a> <img src="https://img.shields.io/github/package-json/v/gnonce/gatsby-starter-gnonce.svg?color=blue"> <img src="https://img.shields.io/badge/license-MIT-brightgreen.svg">
-</p>
+![og-image](https://user-images.githubusercontent.com/6764315/154456671-7b8a9aea-9b69-43de-bd8e-a57b84f53844.png)
 
-## 👩‍💻 Features
+# Deployed environments
 
-- [x] Google Lighthouse optimized (100/100)
-- [x] Search engine optimization
-- [x] Social media and meta tags
-- [x] One icon favicons
-- [x] Manifest support
-- [x] Sitemap support
-- [x] Offline support
-- [x] Easy to use media queries
-- [x] Using Typescript
-- [x] Using styled-components
-- [x] Using Prettier
-- [x] Using tslint
+- Production: https://gnosis-safe.io/
+- Staging: http://safe-team.staging.gnosisdev.com
+- Development: http://safe-team.dev.gnosisdev.com
 
-## 🚀 Quick start
+## Getting Started
 
-1.  **Create a Gatsby site.**
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
```

Comment:
"These instructions will help you..."

Reply:
"[...] help you a copy [...]"?

Reply:
"These instructions will help you get you a copy of the project up and running on your local machine for development and testing purposes."